src/Core/Components/Menu/FluentMenu.razor.cs

```diff
@@ -161,6 +161,18 @@ public bool Open
     [Parameter]
     public int HorizontalThreshold { get; set; } = 200;
 
+    /// <summary>
```

Comment:
You need to add these properties to the `FluentMenuProvider.razor` file.

Reply:
Done